packages/core/src/etherscan.ts

```diff
@@ -311,6 +310,14 @@ export const etherscanVerifySphinxSystem = async (
       let buildInfo: BuildInfo
       if (type === SystemContractType.SPHINX) {
         buildInfo = sphinxBuildInfo
+        // We have to remove `.Darwin.appleclang` to make etherscan verify our contracts which is included in the
+        // compiler version for our contracts. It's unclear why this hasn't been an issue historically.
+        // This could also be an issue when verifying end user contracts, but we haven't seen it yet presumably
+        // because we always recompile the contracts. This could be an argument for continuing to do that.
+        buildInfo.solcLongVersion = buildInfo.solcLongVersion.replace(
+          '.Darwin.appleclang',
+          ''
+        )
```

Comment:
See the comment; we should keep an eye out for this being a problem for users deploying via the DevOps platform especially when we move to only compiling one time.

It's currently impossible for this issue to happen on the DevOps platform since we recompile the contracts ourselves.